README.md

```diff
@@ -131,9 +130,9 @@ config :maru, MongoosePush.Router,
     ]
 ```
 
-This part of configuration relates only to `REST` endpoints that `MongoosePush` exposes. Here you can set bind IP adress (option: `ip`), port and paths to your `HTTP` `TLS` certificates. You should ignore other options unless you know what you're doing or you're going to get to know by reading [maru's documentation](https://maru.readme.io/docs).
+This part of configuration relates only to `HTTP` endpoints exposed by `MongoosePush`. Here you can set a bind IP adress (option: `ip`), port and paths to your `HTTP` `TLS` certificates. You should ignore other options unless you know what you're doing or you're going to know by reading [maru's documentation](https://maru.readme.io/docs).
```

Comment:
You should ignore other options unless you know what you're doing or you're going to follow [maru's documentation](https://maru.readme.io/docs) exactly ?
I still don't like this sentence